server/irmaserver/handle.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/go-chi/chi"
 	"github.com/privacybydesign/gabi"
 	"github.com/privacybydesign/gabi/signed"
-	"github.com/privacybydesign/irmago"
+	irma "github.com/privacybydesign/irmago"
```

Comment:
Hmm it seems this is added by `go` itself. Should I remove this?

Reply:
No, it's fine; we have these changes also in other PRs/commits. Apparently they changed `gofmt` at some point to do this.